staging/src/k8s.io/kube-openapi/pkg/aggregator/aggregator.go

```diff
@@ -474,9 +514,15 @@ func MergeSpecs(dest, source *spec.Swagger) error {
 
 // Clone OpenAPI spec
 func CloneSpec(source *spec.Swagger) (*spec.Swagger, error) {
-	if ret, err := cloner.DeepCopy(source); err != nil {
+	// TODO(mehdy): Find a faster way to clone an spec
```

Comment:
Yes, but it still feels terrible to clone with marshalling/unmarshalling. It is painfully slow (~500ms). I think we can keep the TODO and when we got a better deepcopy, we can use it here.